_topic_map.yml

```diff
@@ -44,6 +44,12 @@ Topics:
   File: architecture
 - Name: Available cluster customizations
   File: customizations
+- Name: Builds
+  File: architecture-component-builds
+- Name: Image Registry
+  File: architecture-component-imageregistry
+- Name: Image streams
+  File: architecture-component-imagestreams
```

Comment:
I had to do this to get my new files to render at all, but it results in them being put directly into the Architecture hierarchy which i don't want.